apps/messages/lib.js

```diff
@@ -153,8 +155,28 @@ exports.getMessageImage = function(msg) {
   return atob("HBKBAD///8H///iP//8cf//j4//8f5//j/x/8//j/H//H4//4PB//EYj/44HH/Hw+P4//8fH//44///xH///g////A==");
 };
 
-exports.getMessageImageCol = function(msg,def) {
-  return {
+function hexTo3Bit(color) {
```

Comment:
Please could you just move the declaration of this inside getMessageImageCol? That way it won't end up using any more RAM